### PR TITLE
docs: Fix command for overwriting iptables on kube-proxy replacement install

### PR DESCRIPTION
**Pull request by Stijn98s**
Please ensure your pull request adheres to the following guidelines:

- [ ] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [ ] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Thanks for contributing!

Updated the restore iptables command because this command wasn't working on Ubuntu 20.10 arm64.

### CODE REVIEW

**Review by qmonnet — COMMENTED** [review]
Thanks! The change looks good, but I would be curious to understand why the previous command wouldn't work. What error did you get?

**Comment by Stijn98s** [comment]
Didn't get a clear error. The changes didn't got applied. These errors occured.

`Can't open /dev/fd/63: No such file or directory`
and
`Warning: iptables-legacy tables present, use iptables-legacy-save to see them`

**Review by qmonnet — APPROVED** [review]
Ah, did you run the command with `sudo`? [Process substitution](https://unix.stackexchange.com/a/279889/70764) might be at play here.

If so, let's update the command indeed to avoid other people to hit the same issue.

**Comment by Stijn98s** [comment]
Yea, added `sudo` to both commands. Still didn't work

**Review by pchaigno — APPROVED** [review]

**Comment by qmonnet** [comment]
Yes, I suppose this is _because_ of the `sudo` that the `<(...)` syntax didn't work. See the answers I linked if you want the details. All good for me!

**Review by aanm — COMMENTED** [review]
@qmonnet but wouldn't we need something like `sudo iptables-save | grep -v KUBE | sudo iptables-restore`? at the very least: `sudo su -c "iptables-save | grep -v KUBE | iptables-restore"`?

**Comment by qmonnet** [comment]
True, we can also add the `sudo` everywhere. This would require updating the other `iptables-save` commands in the document, though.

(At least with the new form in this commit, a missing `sudo` will fail with `Fatal: can't open lock file /run/xtables.lock: Permission denied`, which is a better indication than what we had previously.)

**Comment by brb** [comment]
@Stijn98s Can you try running the previous cmd after entering the root with `sudo -s`?

File: Documentation/gettingstarted/kubeproxy-free.rst [hunk 1]
```diff
@@ -68,7 +68,7 @@ Cilium replacement has been installed:
       # Delete the configmap as well to avoid kube-proxy being reinstalled during a kubeadm upgrade (works only for K8s 1.19 and newer)
       kubectl -n kube-system delete cm kube-proxy
       # Run on each node:
```
Comment (aanm) [inline Documentation/gettingstarted/kubeproxy-free.rst]: This would need something like `"Run on each node with root permissions"`

**Review by aanm — CHANGES_REQUESTED** [review]
We would at least need the following changes

**Comment by aanm** [comment]
@Stijn98s can you rebase the PR? :pray:

**Review by aanm — APPROVED** [review]

**Pull request merged**